packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -337,6 +337,54 @@ void WebView::HandleWebViewMethodCall(const FlMethodCall& method_call,
     } else {
       result->Error("Invalid argument", "No url provided.");
     }
+  } else if (method_name == "loadRequestWithParams") {
+    std::string url;
+    if (!GetValueFromEncodableMap(arguments, "url", &url)) {
+      result->Error("Invalid argument", "No url provided.");
+      return;
+    }
+
+    Ewk_Http_Method ewk_method = EWK_HTTP_METHOD_GET;
+    int32_t method;
+    GetValueFromEncodableMap(arguments, "method", &method);
+    if (method == 1) {  // Post request.
+      ewk_method = EWK_HTTP_METHOD_POST;
+    }
```

Comment:
you should check return value of `GetValueFromEncodableMap` or initialize `method`
```suggestion
      if (GetValueFromEncodableMap(arguments, "method", &method) && method == 1) {  // Post request.
      ewk_method = EWK_HTTP_METHOD_POST;
    }
```

Reply:
Initializing `method` to 1 should be enough.

Reply:
I added exception handling code. Thank you for review.